core-primitives/attestation-handler/src/cert.rs

```diff
@@ -346,6 +371,7 @@ where
 		verify_attn_report(attn_report_raw, pub_k, attestation_ocall)
 	} else {
 		// TODO Refactor state provisioning to not use MURA #1385
+		// TODO DCAP is currently just passed through! SECURITY!!!
```

Comment:
I wasn't aware that our MU RA is insecure for DCAP. Increases the urgency for #1385